src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -526,8 +517,11 @@ private boolean hasReposInFilter() {
         return allReposInFilterExpr.isEmpty();
     }
 
-    private void openPageOfCurrentlySelectedIssue() {
-        TurboIssue issue = getSelectedElement().get().getIssue();
-        ui.getBrowserComponent().showIssue(issue.getRepoId(), issue.getId(), issue.isPullRequest(), false);
+    private TurboIssue getCurrentlySelectedIssue() {
+        return getSelectedElement().get().getIssue();
+    }
+
+    private void openIssuePageMainTab(TurboIssue issue) {
+        ui.getBrowserComponent().showIssueMainTab(issue.getRepoId(), issue.getId(), issue.isPullRequest(), false);
```

Comment:
Since we decided to keep `BrowserComponent.showIssueMainTab`, we won't need to force refresh